cryptus-app/components/wallet_viewer/profile_wallet_viewer/collection_block/collection_block_name/collection_block_name.tsx

```diff
@@ -7,11 +7,11 @@ import s from "./collection_block_name.module.scss";
 //internal imports
 
 
-export default function Collection_block_name(props: { nft: nft}) {
-  const { nft } = props;
+export default function CollectionBlockName(props: { name: string}) {
+  const { name } = props;
   return (
     <div className={s.container}>
-      {nft.collection.toLocaleUpperCase()}
+      <span>{name}</span>
     </div>
   );
 }
```

Comment:
This code is too short (its a oneliner), to have its own folder. Simply copy-paste the code from this folder directly into the parent component.